pkg/kubelet/server/server.go

```diff
@@ -378,14 +376,6 @@ func (s *Server) InstallDefaultHandlers(enableCAdvisorJSONEndpoints bool) {
 		compbasemetrics.HandlerFor(r, compbasemetrics.HandlerOpts{ErrorHandling: compbasemetrics.ContinueOnError}),
 	)
 
-	// deprecated endpoint which will be removed in release 1.20.0+.
-	s.addMetricsBucketMatcher("metrics/resource/v1alpha1")
```

Comment:
what about line 346 of this file? Perhaps needs to be removed a well.

Reply:
Thanks, I missed it. Done now.